scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -24,6 +24,10 @@ Set-Variable EdgeInstallDirectory -Value "$env:ProgramFiles\iotedge" -Option Con
 Set-Variable EdgeDataDirectory -Value "$env:ProgramData\iotedge" -Option Constant
 Set-Variable EdgeServiceName -Value 'iotedge' -Option Constant
 
+Set-Variable ContainersFeatureRegPath -Value 'HKLM:\SYSTEM\CurrentControlSet\Services\vmcompute' -Option Constant
```

Comment:
Is testing the existence of this identical to checking whether `Get-Service vmcompute` succeeds? If so, I think it would be better to do it that way.

Reply:
I believe they both test existence of vmcompute, however, this was suggested to me as the most reliable way to check for the presence of vmcompute on Iot Core.

Reply:
How is it more reliable?

Reply:
Nevermind, "Get-Service" offers same functionality and is cleaner. Fixed.